tests/utils/atom_array_builders.py

```
@@ -0,0 +1,86 @@
+        Occupancy value used when ``with_occupancy=True``.
+    b_factor_value
+        B-factor value assigned to all atoms.
+
+    Returns
+    -------
+    AtomArray
+        Configured atom array for tests.
+    """
+    if chain_ids is not None and res_ids is not None and atom_names is not None:
+        if not (len(chain_ids) == len(res_ids) == len(atom_names)):
+            raise ValueError("chain_ids, res_ids, and atom_names must have matching lengths")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)